examples/acados_matlab_octave/test/test_all_examples.m

```diff
@@ -29,105 +29,113 @@
-%     '../pendulum_dae/example_sim.m';
-%     '../pendulum_on_cart_model/example_closed_loop.m';
-%     '../pendulum_on_cart_model/example_ocp.m';
-%     '../pendulum_on_cart_model/example_ocp_custom_hess.m';
-%     '../pendulum_on_cart_model/example_ocp_param_sens.m';
-%     '../pendulum_on_cart_model/example_ocp_reg.m';
-%     '../pendulum_on_cart_model/example_sim.m';
-%     '../pendulum_on_cart_model/example_solution_sens_closed_loop.m';
-%     '../pendulum_on_cart_model/experiment_dae_formulation.m';
-%     '../race_cars/main.m';
-%     '../simple_dae_model/example_ocp.m';
-%     '../swarming/example_closed_loop.m';
-%     '../swarming/example_ocp.m';
-%     '../swarming/example_sim.m';
-%     '../wind_turbine_nx6/example_closed_loop.m';
-%     '../wind_turbine_nx6/example_ocp.m';
-%     '../wind_turbine_nx6/example_sim.m';
+run_tests();
 
-%     './test_checks.m';
-%     './test_mhe_lorentz.m';
-%     './test_ocp_OSQP.m';
-%     './test_ocp_linear_mass_spring.m';
-%     './test_ocp_pendulum_dae.m';
-%     './test_ocp_pendulum_on_cart.m';
-%     './test_ocp_qpdunes.m';
-%     './test_ocp_simple_dae.m';
-%     './test_ocp_wtnx6.m';
-%     './test_sens_adj.m';
-%     './test_sens_forw.m';
-%     './test_sens_hess.m';
-%     './test_sim_dae.m';
-%     './test_target_selector.m'
-};
+function run_tests()
+   % list the examples you would like to test
 
-test_ok = zeros(1,length(targets));  % keep track of test results
-test_messages = cell(1,length(targets));  % and error messages
+    targets = {
+        '../generic_dyn_disc/disc_dyn_example_ocp.m';
+        '../generic_external_cost/external_cost_example_ocp.m';
+        %'../getting_started/extensive_example_ocp.m';
+        %'../getting_started/minimal_example_closed_loop.m';
+        '../getting_started/minimal_example_sim.m';
+        %'../getting_started/simulink_example.m';
+        %'../getting_started/simulink_example_advanced.m';
+         %'../linear_mass_spring_model/example_closed_loop.m';
+         '../linear_mass_spring_model/example_ocp.m';
+         %'../linear_mass_spring_model/example_sim.m';
+         '../linear_mpc/example_closed_loop.m';
+         '../lorentz/example_mhe.m';
+         %'../masses_chain_model/example_closed_loop.m';
+         '../masses_chain_model/example_ocp.m';
+         %'../pendulum_dae/example_closed_loop.m';  % error while reshaping cost.Vz_0
+         '../pendulum_dae/example_sim.m';
+         %'../pendulum_on_cart_model/example_closed_loop.m';
+         '../pendulum_on_cart_model/example_ocp.m';
+         %'../pendulum_on_cart_model/example_ocp_custom_hess.m';
+         %'../pendulum_on_cart_model/example_ocp_param_sens.m';  % Unable to resolve the name ocp.t_ocp.eval_param_sens.
+         %'../pendulum_on_cart_model/example_ocp_reg.m';
+         %'../pendulum_on_cart_model/example_sim.m';
+         %'../pendulum_on_cart_model/example_solution_sens_closed_loop.m';  % Unable to resolve the name ocp.t_ocp.eval_param_sens.
```

Comment:
There are some comments here, which had been removed in `b862293d7`, e.g. `Unable to resolve...`, probably a merge artifact.
Can you remove them here as well?

And is this comment still an issue?
```
%     '../pendulum_dae/example_closed_loop.m';  % error while reshaping cost.Vz_0
```

Maybe also add a brief comment in the PR description on if all examples work and that we decided not to run all for quick feedback from CI.